<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicParameters</class>
 <widget class="QDialog" name="GraphicParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>325</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>210</width>
    <height>325</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>210</width>
    <height>325</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Graphics</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="MelosimulaRF.qrc">:/Icons/Icons/graphicOptions.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Graphics&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="General">
           <item>
            <widget class="QLabel" name="General_Label">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;General&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="Gen_Ejes_Label">
               <property name="text">
                <string>Axes</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Gen_Ejes_Button">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 0, 4);</string>
               </property>
               <property name="text">
                <string>OFF</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="Gen_FOG_Label">
               <property name="text">
                <string>FOG</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Gen_FOG_Button">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 0, 4);</string>
               </property>
               <property name="text">
                <string>OFF</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="Mod_ContPoints_Label">
               <property name="text">
                <string>Contact Points</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Mod_ContPoints_Button">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 0, 4);</string>
               </property>
               <property name="text">
                <string>OFF</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="Modulos">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="Modulo_Label">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Module&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="Mod_Selection"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="Mod_Trayectoria_Label">
               <property name="text">
                <string>Path</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Mod_Trayectoria_Button">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 0, 4);</string>
               </property>
               <property name="text">
                <string>OFF</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="Mod_Ejes_Label">
               <property name="text">
                <string>Axes</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Mod_Ejes_Button">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 0, 4);</string>
               </property>
               <property name="text">
                <string>OFF</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="Mod_CADCub_Label">
               <property name="text">
                <string>CAD/Box</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Mod_CADCub_Button">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(42, 210, 0);</string>
               </property>
               <property name="text">
                <string>CAD</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="MelosimulaRF.qrc"/>
 </resources>
 <connections/>
</ui>
